app/src/main/java/ar/edu/unlam/mobile/scaffolding/data/datasources/local/dao/UserDao.kt

```diff
@@ -1,25 +1,26 @@
-package ar.edu.unlam.mobile.scaffolding.data.database.dao
+package ar.edu.unlam.mobile.scaffolding.data.datasources.local.dao
 
 import androidx.room.Dao
+import androidx.room.Delete
 import androidx.room.Insert
 import androidx.room.OnConflictStrategy
 import androidx.room.Query
-import ar.edu.unlam.mobile.scaffolding.data.database.entities.UserEntity
+import ar.edu.unlam.mobile.scaffolding.data.datasources.local.entities.UserEntity
 
 @Dao
 interface UserDao {
     private companion object {
         const val ID = 1
     }
 
-    @Query("SELECT * FROM user_table")
+    @Query("SELECT * FROM user_table WHERE id = ID")
```

Comment:
```suggestion
    @Query("SELECT * FROM events WHERE id=:id ")
```

Si quieren obtener un usuario por si ID,
recibir el id por parámetro:
```kotlin
suspend fun getUser(id: String): UserEntity?
```